Documentation/trying-out-rkt.md

```diff
@@ -16,6 +16,8 @@ cd rkt-v1.8.0
 ./rkt help
 ```
 
+Note that in order to fully utilize rkt there is a couple of additional steps to be performed. Therefore we highly recommend installing rkt using your local package manager. If rkt is not available for your Linux distribution please consider to try rkt using Vagrant as described below.
```

Comment:
This is a bit contradictory with lines 8-10 now. Can we merge them together? Something along the lines of starting with 
"It's recommended to install rkt using your local package manager for the best experience. If rkt is not available for your Linux distribution, or you want to use the latest version, ... ....." @joshix or co can probably advise on the best route